CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fix command line sample to install latest node dependencies in migration guides.
 - Fix the broken markdown URLs due to this [storybook issue](https://github.com/storybookjs/storybook/issues/9005) by replacing markdown links with HTML `<a>` tag.
+- Clean up the wrong statement in stories of `LocalVideoProvider` and `useLocalVideo`.
```

Comment:
Would be good to know about the actual change: "Clarify `setIsVideoEnabled` usage in `LocalVideoProvider` and `useLocalVideo` docs.

Reply:
Good idea, updated!